framework/ntru.sage

```diff
@@ -0,0 +1,92 @@
+# Code adapted from:
+# https://github.com/kpatsakis/NTRU_Sage/blob/master/ntru.sage
+# Under GPL 2.0 license.
+
+from time import time
+R.<x>  =  ZZ['x'];
```

Comment:
Using global variables can lead to problems if the user defines R or x after loading the framework but before using NTRUEncrypt. A possible solution is to define them as class attributes.